relayer/relays/ethereum/parachain-writer.go

```diff
@@ -212,20 +188,7 @@ func (wr *ParachainWriter) WritePayload(ctx context.Context, payload *ParachainP
 		return err
 	}
 
-	onProcessed := func() error {
-		// Confirm that the header import was successful
-		header := payload.Header.HeaderData.(ethereum.Header)
-		hash := header.ID().Hash
-		imported, err := wr.queryImportedHeaderExists(hash)
```

Comment:
you don't think this is needed anymore? so its basically fire and forget?

Reply:
Yeah good question. The problem is that it wasn't really working 30% of time. In the sense that the extrinsic was successfully included in a block, but the query failed. Was causing a huge number of spurious restarts.

So I want to go back to basics and then potentially add it back at a later stage.

Reply:
Looks like the import check is needed for healthy operation. So I've added it back, but with tweaks - we only perform the check if the extrinsic was finalized, rather than merely being included in a block. This seems to prevent the spurious import check errors.